src/App.jsx

```diff
@@ -14,70 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const clearSelection = () => {
+    setSelectedGood('');
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isSelected = good => selectedGood === good;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? (
+            <>
+              {`${selectedGood} is selected`}
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={clearSelection}
+              />
+            </>
+          )
+          : ('No goods selected')
+        }
+      </h1>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => (
+            <tr
+              // eslint-disable-next-line react/no-array-index-key
+              key={index}
```

Comment:
you can pass `good` here
```suggestion
              key={good}
```